### PR TITLE
Testing the merge procedure

### PR DESCRIPTION
**Pull request by Christo81**
Test test nada blah blah

### CODE REVIEW

File: source/ImageProcessor.cpp [hunk 1]
```diff
@@ -25,20 +25,54 @@ ImageProcessor::ImageProcessor(double cannyLowThreshold,
 	element = cv::getStructuringElement(cv::MORPH_RECT, cv::Size(3, 3), cv::Point(2, 2));
 }
 
+//cv::Mat ImageProcessor::LaneFilter(cv::Mat &inputMat, cv::Scalar &medianLaneColor)
```
Comment (mattwhiteEB) [inline source/ImageProcessor.cpp:28]: is there any reason to leave all this commented code in? (I'd prefer it gets cleaned up). Same for most other comments.

**Comment by TallPatrick** [comment]
yup, that's a code change.

**Review by mattwhiteEB — COMMENTED** [review]
With switching to using a SVM, this review / pull request can be dropped / rejected now, correct?

**Comment by Christo81** [comment]
I wouldn't remove anything before having something else (machine learning) that works at least a little.

**Comment by Christo81** [comment]
Here is the code for the LBP SVM for whoever wants to take a look. It still needs a lot of work and changes though.
https://gist.github.com/Christo81/d83148ead103d9d9d1c561dc3f4ca012#file-gistfile1-txt-L249

**Pull request closed without merging**